### PR TITLE
refactor directory structure

### PR DESCRIPTION
**Pull request by shiodat**
Related to issue #287, I changed the directory structure to follow the table of contents.

### CODE REVIEW

**Comment by shiodat** [comment]
I fixed the conflicts and refactor English docs.

**Comment by kmaehashi** [comment]
* I think it is better to name the directory index file as ``index.rst``, e.g., ``/ja/overview/index.rst`` instead of ``/ja/overview/summary.rst`` so that users can access it like ``http://jubat.us/ja/overview/``.

**Comment by kmaehashi** [comment]
@shiodat
* Please update methods page in English as discussed.
* Please resolve the conflict.

**Comment by shiodat** [comment]
@kmaehashi 
- I renamed the index files , e.g. `/ja/jubaql/jubaql.rst` <- `/ja/jubaql/index.rst`.
- I removed the Japanese descriptions at English methods page `en/method.rst`.
- I resolved the conflicts.

File: source/en/fv_convert.rst [hunk 1]
```diff
@@ -534,7 +534,7 @@ It depends on "type" how to specify weight and name features.
     Value         Meaning
     ============= =====================
     ``"num"``     Use given number itself as weight.
-    ``"log"``     Use natural logarithm of given number as weight. If the number is not positive, weight is 0.
+    ``"log"``     Use logarithm of given number as weight. If the number is not positive, weight is 0.
```
Comment (kmaehashi) [inline source/en/fv_convert.rst]: Degraded when resolving conflict?

File: source/ja/fv_convert.rst [hunk 2]
```diff
@@ -630,7 +630,7 @@ string_rulesなどと同様、複数の規則を羅列する。
    値           意味
    ============ =====================
    ``"num"``    与えられた数値をそのまま重みに利用する。
-   ``"log"``    与えられた数値の自然対数を重みに利用する。但し、数値が1以下の場合は0とする。
+   ``"log"``    与えられた数値の対数を重みに利用する。但し、数値が1以下の場合は0とする。
```
Comment (kmaehashi) [inline source/ja/fv_convert.rst]: Degraded when resolving conflict?

File: source/en/fv_convert.rst (hunk 1 above)
Comment (shiodat) [inline source/en/fv_convert.rst]: I resolved conflicts again and fix the degradation in `fv_converter`.

File: source/ja/fv_convert.rst (hunk 2 above)
Comment (shiodat) [inline source/ja/fv_convert.rst]: I resolved conflicts again and fix the degradation in `fv_converter`.

**Comment by shiodat** [comment]
I resolved conflicts again and fix the degradation in `fv_converter`.
I used the old version of `fv_converter` and degredation occured, sorry.

**Comment by kmaehashi** [comment]
:+1: Thank you!

**Pull request merged**